controls/V-72039.rb

```diff
@@ -1,87 +1,62 @@
-unauthorized modification.
+  tag "cci": ["CCI-000318", "CCI-000368", "CCI-001812", "CCI-001813",
+"CCI-001814"]
+  tag "documentable": false
+  tag "nist": ["CM-3 f", "CM-6 c", "CM-11 (2)", "CM-5 (1)", "CM-5 (1)", "Rev_4"]
+  tag "check": "Verify that all system device files are correctly labeled to
+prevent unauthorized modification.
 
-List all device files on the system that are incorrectly labeled with the following
-commands:
+List all device files on the system that are incorrectly labeled with the
+following commands:
 
-Note: Device files are normally found under \"/dev\", but applications may place
-device files in other directories and may necessitate a search of the entire system.
+Note: Device files are normally found under \"/dev\", but applications may
+place device files in other directories and may necessitate a search of the
+entire system.
 
-#find /dev -context *:device_t:* \\( -type c -o -type b \\) -printf \"%p %Z\
-\"
+#find /dev -context *:device_t:* \\( -type c -o -type b \\) -printf \"%p %Z\n\"
 
-#find /dev -context *:unlabeled_t:* \\( -type c -o -type b \\) -printf \"%p %Z\
+#find /dev -context *:unlabeled_t:* \\( -type c -o -type b \\) -printf \"%p %Z\n
 \"
 
 Note: There are device files, such as \"/dev/vmci\", that are used when the
-operating system is a host virtual machine. They will not be owned by a user on the
-system and require the \"device_t\" label to operate. These device files are not a
-finding.
+operating system is a host virtual machine. They will not be owned by a user on
+the system and require the \"device_t\" label to operate. These device files
+are not a finding.
 
-If there is output from either of these commands, other than already noted, this is
-a finding."
-  tag "fix": "Run the following command to determine which package owns the device
-file:
+If there is output from either of these commands, other than already noted,
+this is a finding."
+  tag "fix": "Run the following command to determine which package owns the
+device file:
 
 # rpm -qf <filename>
 
 The package can be reinstalled from a yum repository using the command:
 
 # sudo yum reinstall <packagename>
 
-Alternatively, the package can be reinstalled from trusted media using the command:
+Alternatively, the package can be reinstalled from trusted media using the
+command:
 
 # sudo rpm -Uvh <packagename>"
+  tag "fix_id": "F-78391r1_fix"
 
   # @todo - check for host vm device files as mentioned and pass if found
-  describe command('find /dev -context *:device_t:* \( -type c -o -type b\) -printf "%p %Z"') do
-    its('stdout.strip') { should match /^$/ }
-  end
-  describe command('find /dev -context *:unlabeled_t:* \( -type c -o -type b\) -printf "%p %Z"') do
-    its('stdout.strip') { should match /^$/ }
+  findings = Set[]
+  findings = findings + command('find / -context *:device_t:* \( -type c -o -type b \) -printf "%p %Z\n"').stdout.split("\n")
```

Comment:
This should be marked as dangerous as any global `find` could bring a system to its knees.

Reply:
@trevor-vaughan I'm not sure exactly what you mean by "mark as dangerous"

Reply:
@trevor-vaughan What does it mean to mark it as dangerous?